### ISSUE TITLE
docker image for photoprism preview

### ISSUE THREAD

**Issue by totti4ever** [thielepaul/photoprism-mobile#131]
Hey, I was wondering whether there is a chance to combine the latest development regarding face detection and still use your app?

kind regards

**Comment by thielepaul** [comment]
I rebased my branch onto master yesterday, so the `thielepaul/photoprism:db-api` docker image should contain most of the new features now

**Comment by totti4ever** [comment]
hmmm, it worked, but only once. Could it be that the auto build needs an additional trigger on the upstream master?

**Comment by thielepaul** [comment]
yes that is right I still have to manually rebase my branch on the current master, not sure if this could be automated. At least not if there are conflicts between the commits on both branches

**Comment by totti4ever** [comment]
Ah, I see. Makes sense. Just manually manipulating the official Docker image wouldn't work, would it? 

Possible is quite a lot of things I suppose, using GitHub actionse. But only worth it if little risk of conflicts. Do you expect many?

**Comment by thielepaul** [comment]
yes, could be possible, but I think it is not worth the effort. There are conflicts quite often because of the database query changes. Additionally something could break after a automatic rebase and so overall I think doing this manually makes more sense.

**Comment by totti4ever** [comment]
Would you mind, doing it now? There was a major release

**Comment by thielepaul** [comment]
yes, I already updated it an hour ago or so :)

**Comment by totti4ever** [comment]
Would you mind, doing it again? 🙂

**Comment by thielepaul** [comment]
i did another rebase, docḱer image should be built automatically. Let's hope everything still works with the new image

**Comment by djroy25** [comment]
Hi,

I am running photoprism for a while now.. I want to update to the latest version and use your app..

Can you guys please provide any guidance steps for updating from photoprism to your docker and new app? As it seems not straight forward..

Currently in docker I see 2 pods.. 1 maria db and the other photopirsm.

Thank you

**Comment by pedrommvv** [comment]
**I guess** you just need to point mariadb to this image to `thielepaul/photoprism:db-api`

Edit:
Not at all, you use that image to the service `photoprism`. I read the README file, I wrongly assumed the backend was the mariadb service 💯 and it's works 🥇
